lib/watchbuild/runner.rb

```diff
@@ -67,10 +68,17 @@ def notification(build, minutes)
       end
 
       url = "https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/ra/ng/app/#{@app.apple_id}/activity/ios/builds/#{build.train_version}/#{build.build_version}/details"
-      TerminalNotifier.notify('Build finished processing',
-                              title: build.app_name,
-                              subtitle: "#{build.train_version} (#{build.build_version})",
-                              execute: "open '#{url}'")
+
+      if !ENV['SLACK_URL'].empty?
+        notify_slack(
+          "App Store build #{build.train_version} (#{build.build_version}) has finished processing in #{minutes} minutes",
+          ENV['SLACK_URL'])
+      else
+        TerminalNotifier.notify('Build finished processing',
```

Comment:
Potentially create a `notify_terminal` function and move this logic into there? (should also include the url generation and `require 'terminal-notifier`